Gems/ROS2Sensors/Code/Include/ROS2Sensors/Camera/CameraConfigurationRequestBus.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Math/Matrix3x3.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/RTTI/RTTI.h>
+#include <ROS2Sensors/Camera/CameraSensorConfiguration.h>
+
+namespace ROS2Sensors
+{
+    //! Interface that allows to get and set Camera sensor's configuration.
+    class CameraConfigurationRequest : public AZ::EBusTraits
+    {
+    public:
+        using BusIdType = AZ::EntityId;
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+
+        //! Returns the current configuration of the component.
+        virtual const CameraSensorConfiguration GetConfiguration() const = 0;
+
+        //! Sets the configuration of the component.
+        //! Each component should handle the configuration change without fully reinitializing the ROS2 publisher.
+        //! This will allow to change the configuration of the component at runtime.
+        //! @param configuration The new configuration to set.
+        virtual void SetConfiguration(const CameraSensorConfiguration& configuration) = 0;
+
+        //! Returns the intrinsic calibration matrix of the camera as:
+        //!  [fx  0 cx]
+        //!  [ 0 fy cy]
+        //!  [ 0  0  1]
+        //! where:
+        //!  - fx, fy : the focal lengths in meters
+        //!  - cx, cy : principal point in pixels.
+        virtual AZ::Matrix3x3 GetCameraMatrix() const = 0;
```

Comment:
```suggestion
        virtual AZ::Matrix3x3 GetCameraMatrix() = 0;
```
There is problem to reflect with behavior context with const method

Reply:
It applies to all setters in this PR.